cmd/publicize/README.md

```diff
@@ -0,0 +1,17 @@
+# Publicize
+
+This tool is an external prow plugin. Its purpose is to merge the commit histories of two repositories.
+It responses in a `/publicize` command only in merged pull requests.
```

Comment:
"responds to"

Reply:
I would really like to see a better, more detailed documentation about what the tool is really doing. Can we get a simple workflow like:

1. User enters `/publicize` in a PR that is already merged
2. The plugin fetches the current remote of (which branch?)
3. The plugin merges what into what
4. The plugin files a PR / directly pushes (what hapepns when a pushj fails?)